src/mongocxx/test/pool.cpp

```diff
@@ -96,6 +97,7 @@ TEST_CASE(
     REQUIRE(interposed.crl_file == crl_file);
     REQUIRE(interposed.weak_cert_validation == allow_invalid_certificates);
 }
+#endif
```

Comment:
It might be tricky because we would need to build the C driver twice. Another way would be to make it so that building with SSL support was a configuration option for us. Could be:
- Yes: use SSL, error if underlying C driver does not offer it
- No: Don't use SSL, even if underlying C driver does offer it
- Auto: SSL if C driver SSL.

Do you think that is worth adding to this commit? That would make it trivial to do a no-SSL build of the C++ driver without needing two C driver builds.